sapi/cli/php_cli_server.c

```diff
@@ -2593,11 +2687,13 @@ int do_cli_server(int argc, char **argv) /* {{{ */
 			memmove(buf, "unknown time, can't be fetched", sizeof("unknown time, can't be fetched"));
 		}
 
-		printf("PHP %s Development Server started at %s"
-				"Listening on http://%s\n"
-				"Document root is %s\n"
-				"Press Ctrl-C to quit.\n",
-				PHP_VERSION, buf, server_bind_address, document_root);
+#if HAVE_FORK
+        printf("[%lu] PHP %s Development Server (http://%s) started at %s",
```

Comment:
I think this should be %ld, unless there's a magic constant specifier for pid_t ?